main.js

```diff
@@ -0,0 +1,105 @@
+const currentWeatherData = {};
+const forecastData = {};
+const daysOfTheWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+const apiKey = `c1300dccedd35aac7b51e962f9c75288`;
+
+$('#search').on('click', function() {
+  const city = $('#city-name').val();
+  $('#city-name').val('');
+
+  if (city) {
+    fetchCurrent(city);
+    fetchForecast(city);
+  }
+});
+
+$('form').on('keypress', function(e) {
+  if (e.keyCode === 13) {
+    e.preventDefault();
+
+    const city = $('#city-name').val();
+    $('#city-name').val('');
+  
+    if (city) {
+      fetchCurrent(city);
+      fetchForecast(city);
+    }
+  }
+});
+
+const fetchCurrent = (cityName) => {
+  $.ajax({
+    method: "GET",
+    dataType: "JSON",
+    url: `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=imperial&appid=${apiKey}`,
+    success: function(data) {
+      addToCurrentWeatherData(data);
+    },
+    error: function(jqXhr, textStatus, errorMessage) {
+      console.log(textStatus);
+    }
+  });
+};
+
+const fetchForecast = (cityName) => {
+  $.ajax({
+    method: "GET",
+    dataType: "JSON",
+    url: `https://api.openweathermap.org/data/2.5/forecast?q=${cityName}&units=imperial&appid=${apiKey}`,
+    success: function(data) {
+      addToForecastData(data);
+    },
+    error: function(jqXhr, textStatus, errorMessage) {
+      console.log(textStatus);
+    }
+  });
+};
+
+const addToCurrentWeatherData = data => {
+  currentWeatherData.currentCity = data.name || null;
+  currentWeatherData.currentTemp = Math.round(data.main.temp) || null;
+  currentWeatherData.currentCondition = data.weather[0].main || null;
+  currentWeatherData.currentIcon = data.weather[0].icon || null;
+
+  render();
+};
+
+const addToForecastData = data => {
+  forecastData.day1Temp = Math.round(data.list[7].main.temp); 
```

Comment:
`forecastData` feel bloated - it's an object with 20 properties? It would probably be more ideal for it to be an array of 5 objects.